02-dz0/01-semantic-mockup/style.css

```diff
@@ -0,0 +1,84 @@
+*
+{
```

Comment:
Обычно используется такое форматирование:
```suggestion
.any_selector {
    font-family: Arial, Helvetica, sans-serif;
}
```

Reply:
Исправил